src/playground/standalone-player.jsx

```diff
@@ -0,0 +1,105 @@
+import VirtualMachine from 'scratch-vm';
+import RenderWebGL from 'scratch-render';
+import AudioEngine from 'scratch-audio';
+import {SVGRenderer, BitmapAdapter} from 'scratch-svg-renderer';
+import storage from '../lib/storage';
+
+// This file is an example of how to create a standalone, full screen
+// minimal scratch player without the editor view.
+
+const EXAMPLE_PROJECT = 10128067; // 'Make it Dance' Project
```

Comment:
I think we want the example project to follow the default the other scratch-gui pages use. The Scratch Cat default project id 0.